handle_channel_data.py

```diff
@@ -355,26 +292,24 @@ def get_and_update_data_firebase(is_update=True):
     root = db.reference()
 
     # retrieve current videos
-    if is_update:
-        videos = root.child('video_data').get()
-        video_data = update_videos_from_channel(page_token=None, data=videos)
-    else:
-        video_data = get_videos_from_channel(page_token=None)
-
-    processed_data = process_grade_data(data=video_data)
-    processed_data = process_climber_data(data=processed_data)
-    processed_data = process_zone_data(data=processed_data)
-
-    video_data = {
-        'date': str(date.today()),
-        'items': processed_data
-    }
-
-    # Update videos
+   #if is_update:
+   #    videos = root.child('video_data').get()
+   #    video_data = update_videos_from_channel(data=videos)
+   #else:
+   #    video_data = get_videos_from_channel(page_token=None)
+   #
+   #processed_data = process_grade_data(data=video_data)
+   #processed_data = process_climber_data(data=processed_data)
+   #processed_data = process_zone_data(data=processed_data)
+   #
+   #video_data = {
+   #    'date': str(date.today()),
+   #    'items': processed_data
+   #}
+    
+    video_data = get_data_local()
```

Comment:
@juangallostra Can you do some further checking if this could be possible? I think the code is a lot clearer if there is only one option (working with stored jsons) insted of two (working with stored jsons or in-memory ones). It got me some time to understand what the code was doing. The files do not need to be saved, and it shouldn't use more memory because they already exist, so I think it's worth a shot.

Reply:
will do!